.eslintrc

```diff
@@ -127,7 +127,7 @@
         },
         "project": ["./tsconfig.json"]
       },
-      "plugins": ["@typescript-eslint", "jest"],
+      "plugins": ["@typescript-eslint", "jest", "no-restricted-imports"],
```

Comment:
I think there's no need for the `eslint-plugin-no-restricted-imports` considering that `no-restricted-imports` is a rule built in eslint
https://eslint.org/docs/latest/rules/no-restricted-imports#version

I've tested it and it seems to work without adding that plugin

Reply:
Yeah I wrongly thought it was only added in v8, I think I had the syntax incorrect when I tested if it was built in as it does work, I'll  update it soon 🤦